gunicorn/datadog_checks/gunicorn/gunicorn.py

```diff
@@ -129,16 +130,16 @@ def _get_master_proc_by_name(self, name, tags):
                 self.log.debug("Cannot read information from process %s: %s", p.name(), e, exc_info=True)
         if len(master_procs) == 0:
             # process not found, it's dead.
+            msg = "No gunicorn process with name %s found" % name
             self.service_check(
                 self.SVC_NAME,
                 AgentCheck.CRITICAL,
                 tags=['app:' + name] + tags,
-                message="No gunicorn process with name %s found" % name,
+                message=msg,
             )
-            raise GUnicornCheckError("Found no master process with name: %s" % master_name)
-        else:
-            self.log.debug("There exist %s master process(es) with the name %s", len(master_procs), name)
-            return master_procs
+            self.warning(msg)
```

Comment:
```suggestion
            self.log.warning(msg)
```

Reply:
I think we keep it as `self.warning` similar to the [process check](https://github.com/DataDog/integrations-core/blob/7.25.x/process/datadog_checks/process/process.py#L454); useful to show in status log when troubleshooting.

```=========
Collector
=========

  Running Checks
  ==============

    gunicorn (1.14.0)
    -----------------
      Instance ID: gunicorn:c47c2a872f03460d [WARNING]
      Configuration Source: file:/etc/datadog-agent/conf.d/gunicorn.d/conf.yaml
      Total Runs: 1
      Metric Samples: Last Run: 0, Total: 0
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 1, Total: 1
      Average Execution Time : 14ms
      Last Execution Date : 2021-01-11 00:20:22.000000 UTC
      Last Successful Execution Date : 2021-01-11 00:20:22.000000 UTC

      Warning: No gunicorn process with name missing_master_proc found
```